zstacklib/zstacklib/gpu/vendors/template.py

```
@@ -0,0 +1,435 @@
+    # ==========================================================================
+    # Vendor Identification (REQUIRED)
+    # ==========================================================================
+    
+    # Unique vendor name - used for logging and registry
+    # Example: "NVIDIA", "AMD", "Huawei", "Enflame"
+    VENDOR_NAME = "NewVendor"
+    
+    # PCI Vendor IDs (from lspci -n)
+    # Find with: lspci -nn | grep -i <vendor_keyword>
+    # Example: {"10de"} for NVIDIA, {"1002"} for AMD
+    VENDOR_IDS = {"xxxx"}  # TODO: Set correct vendor ID
```

Ruff (0.14.14)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)